src/domain/login/PasswordLoginViewModel.js

```diff
@@ -57,7 +85,13 @@ export class PasswordLoginViewModel extends ViewModel {
                 break;
         }
         if (error) {
-            this._showError(error);
+            if (usernameChanged) {
+                console.log(error);
+                this._options.showConnectionError(error);
```

Comment:
Why do we have two different ways of showing errors here?

Reply:
Surely a miss from a previous commit